lib/superstring/faker-map.js

```diff
@@ -0,0 +1,146 @@
+module.exports = {
```

Comment:
We need a sanity test against this map to account for any breaking change in faker API.
Also, to make sure there are no duplicates for the same faker `random.` method.